gradle_plugin/template/gradle.properties

```diff
@@ -29,15 +29,19 @@ attachmentConfigPath =
 neededPermissions = # Optional, list of permission names that the test requires, separated by comma. Example: "android.Permission1, android.Permission2"
 
 # Optional, list of actions that the test will operate on the device, content should be in format of a JSON string.
-# (In yml config file, this param is assigned with value directly by hierarchy)
+# (In yml config file, this param is assigned with value directly by hierarchy, see more in: https://github.com/microsoft/HydraLab/blob/main/gradle_plugin/template/testSpec.yml)
 # 1. Current support actions for during setting up and tearing down, keys of the first level can be selected from: setUp | tearDown. The value of them should both be a JSON array.
 # 2. Device types, as the value of key "deviceType", currently can be selected from a supporting list ["Android", "iOS", etc...]. This key is optional.
 # 3. Method types, as the value of key "method", currently can be selected from a supporting list ["setProperty", "setDefaultLauncher", "backToHome", "changeGlobalSetting", "changeSystemSetting"];
 # 4. Provide corresponding params for target methods.
-# See more details in Hydra Lab wiki (section: TBD).
 # Example: "{\"setUp\":[{\"deviceType\":\"Android\",\"method\":\"setProperty\",\"args\":[\"value A\", \"value B\"]}, {...}, {...}], \"tearDown\":[{\"method\":\"backToHome\",\"args\":[]}, {...}, {...}]}"
 deviceActions =
 
+# Optional, list of inspection strategies used for performance test config, content should be in format of a JSON string.
+# (In yml config file, this param is assigned with value directly by hierarchy, see more in: https://github.com/microsoft/HydraLab/blob/main/gradle_plugin/template/testSpec.yml)
+# Example: "[{"strategyType":"TEST_SCHEDULE","interval":300,"intervalUnit":"MILLISECONDS","inspection":{"inspectorType":"INSPECTOR_ANDROID_BATTERY_INFO","appId":"com.microsoft.appmanager","deviceIdentifier":"","description":"test schedule","inspectionKey":"","isReset":false}},{"strategyType":"TEST_LIFECYCLE","when":["TEST_STARTED","TEST_SUCCESS","TEST_FAILURE","TEST_RUN_STARTED","TEST_RUN_FINISHED"],"interval":0,"inspection":{"inspectorType":"INSPECTOR_ANDROID_BATTERY_INFO","appId":"com.microsoft.appmanager","deviceIdentifier":"","description":"test schedule","inspectionKey":"","isReset":false}}]"
```

Comment:
We are not expected to expose the `deviceIdentifier`, `inspectionKey` and `isReset` to the user, which will cause confusion. Those three params will be automatically configured by the HydraLab. Can we remove them from the `inspectionStrategiesStr`?